tests/time_tests/src/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# add dummy `time_tests` target combines all time tests
+add_custom_target(time_tests)
+
+# Build test from every source file matchs *-pipeline.cpp.
+# Test target name is source file name without extension.
```

Comment:
@asomsiko should we extend target name with some unique prefix to determine belonging to `time-tests`? At the moment `infer-pipeline` isn't clear for an outside developer